client/src/js/components/bhDebtorGroupHistory.js

```diff
@@ -0,0 +1,33 @@
+angular.module('bhima.components')
+  .component('bhDebtorGroupHistory', {
+    templateUrl : 'modules/templates/bhDebtorGroupHistory.tmpl.html',
+    controller : bhDebtorGroupHistoryController,
+    transclude : true,
+    bindings : {
+      debtorUuid : '@?',
+      limit : '<',
+    },
+  });
+
+bhDebtorGroupHistoryController.$inject = ['DebtorGroupService', 'NotifyService'];
+/**
+ * Debtor group history component
+ *
+ */
+function bhDebtorGroupHistoryController(DebtorGroupService, Notify) {
+  const $ctrl = this;
+
+  // fired at the beginning
+  $ctrl.$onInit = () => {
+    $ctrl.limit = $ctrl.limit || 5;
+    loadHistory();
+  };
```

Comment:
Let's also put a [`$onChanges`](https://docs.angularjs.org/guide/component#component-based-application-architecture) binding so that we reload the history when the `debtorUuid` changes.